packages/SwingSet/src/liveslots/virtualObjectManager.js

```diff
@@ -419,11 +419,9 @@ export function makeVirtualObjectManager(
     if (typeof obj !== 'object') {
       return 'not';
     }
-    if (Object.getOwnPropertySymbols(obj).length !== 0) {
-      return 'not';
-    }
     let established;
-    for (const [_name, value] of Object.entries(obj)) {
+    for (const prop of Reflect.ownKeys(obj)) {
```

Comment:
According to my crib sheet, `Reflect.ownKeys` returns non-enumerable properties, whereas `Object.entries` ignored them. But I think that's fine here. It'll cause a tiny behavior change: if someone provides a behavior object with non-enumerable properties, then previously they'd be ignored entirely, and now they'll be copied onto the Representative (and become enumerable), which doesn't seem like a problem to me.